### ISSUE TITLE
F-Droid build failed

### ISSUE THREAD

**Issue by linsui** [ivpn/android-app#110]
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:kaptReleaseKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.gradle.internal.KaptExecution
   > java.lang.reflect.InvocationTargetException (no error message)
```

Could you please take a look? Thanks!

**Comment by theanonymousexyz** [comment]
Hey, any update on this? The F-Droid version is still on v2.6.0.

**Comment by jurajhilje** [comment]
It's in our pipeline, we plan to resolve it soon (next few weeks).

**Comment by jurajhilje** [comment]
Fixed in **v2.8.2**.
